src/QMCHamiltonians/CoulombPBCAA.h

```diff
@@ -75,6 +75,8 @@ struct CoulombPBCAA : public OperatorBase, public ForceBase
   Vector<ComplexType> del_eikr;
   /// Flag for whether to compute forces or not
   bool ComputeForces;
+  /// Flag for whether to use quasi-2D Ewald
+  bool quasi2d;
```

Comment:
This should be const with its value being initialized by the constructor member initializer lists.

Reply:
I guess you changed the initialization but forgot to add const.